<?xml version="1.0"?>

<launch>
    <arg name="des_x" /> 
    <arg name="des_y" />

    <param name="des_pos_x" value="$(arg des_x)" /> 
    <param name="des_pos_y" value="$(arg des_y)" />

    <node pkg= "motion_plan" type="srv_server_go_to_point.py" name="go_to_point" output="screen"/>
    <node pkg= "motion_plan" type="srv_server_wall_follower.py" name="wall_follower" output="screen"/>
    <node pkg= "motion_plan" type="bug0.py" name="bug_0" output="screen"/>
</launch>